python/python/glide/async_commands/core.py

```diff
@@ -831,6 +831,86 @@ async def hkeys(self, key: str) -> List[str]:
         """
         return cast(List[str], await self._execute_command(RequestType.Hkeys, [key]))
 
+    async def hrandfield(self, key: str) -> Optional[str]:
+        """
+        Returns a random field name from the hash value stored at `key`.
+
+        See https://redis.io/commands/hrandfield/ for more details.
+
+        Args:
+            key (str): The key of the hash.
+
+        Returns:
+            Optional[str]: A random field name from the hash stored at `key`.
+            If the hash does not exist or is empty, None will be returned.
+
+        Examples:
+            >>> await client.hrandfield("my_hash")
+                "field1"  # A random field name stored in the hash "my_hash".
+        """
+        return cast(
+            Optional[str], await self._execute_command(RequestType.HRandField, [key])
+        )
+
+    async def hrandfield_count(self, key: str, count: int) -> List[str]:
+        """
+        Retrieves random field names from the hash value stored at `key`.
+
+        See https://redis.io/commands/hrandfield/ for more details.
+
+        Args:
+            key (str): The key of the hash.
+            count (int): The number of field names to return.
+                If `count` is positive, returns unique elements.
+                If negative, allows for duplicates.
+
+        Returns:
+            List[str]: A list of random field names from the hash.
+            If the hash does not exist or is empty, the response will be an empty list.
+
+        Examples:
+            >>> await client.hrandfield_count("my_hash", -3)
+                ["field1", "field1", "field2"]  # Non-distinct, random field names stored in the hash "my_hash".
+            >>> await client.hrandfield_count("non_existing_hash", 3)
+                []  # Empty list
+        """
+        return cast(
+            List[str],
+            await self._execute_command(RequestType.HRandField, [key, str(count)]),
+        )
+
+    WITH_VALUES: str = "WITHVALUES"
```

Comment:
we dont use that for python, its better to not add that

Reply:
Coding best practice recommend using constants instead of magic numbers/strings to avoid mistakes.

Reply:
I'd recommend that we use constants.  
But it would be better to include them in another file (rather than buried here in the middle of core.py.  
Common practice is to create a separate file (like command_constants.py) to centralize these constants.  I'm not sure you're interested in creating one @aaron-congo?

Reply:
I can create one if we want one, @shohamazon what do you think?